.gitignore

```diff
@@ -8,3 +8,4 @@ __pycache__
 /rpmbuild
 
 /tools/appsre-ansible/inventory
+/.idea
```

Comment:
Sure, but maybe a different commit, thanks! ;)

Reply:
Absolutely. I am hoping for some opinions on the actual logging work tho :-)